### ISSUE TITLE
move `react` from `dependencies` to `peerDependencies`

### ISSUE THREAD

**Issue by ksmth** [Lapple/react-json-inspector#6]
I've had trouble integrating - once I removed the modules local copy of react, everything was fine. :)

**Comment by Benoss** [comment]
Same issue here.
This is because react is fixed to 0.13.0 in dependencies this is reimporting another version of react in my project
This is the strange error message I have

```
Only a ReactOwner can have refs. This usually means that you're trying to add a ref to a component that doesn't have an owner (that is, was not created inside of another component's `render` method). Try rendering this component inside of a new top-level component which will hold the ref.
```

Ref: http://stackoverflow.com/questions/28519287/what-does-only-a-reactowner-can-have-refs-mean

**Comment by Lapple** [comment]
Released as `5.0.1`.
